### ISSUE TITLE
HomeAssistant Add-in

### ISSUE THREAD

**Issue by awraeven** [fboaventura/dckr-mrtg#29]
Is there any chance that you will enable this very wonderfull container as a Home Assistant add-in?
HA supports SNMP, but I would loose all my stats of MRTG.

**Comment by fboaventura** [comment]
@ExquisiteMetropolis I'll look into this.

**Comment by awraeven** [comment]
@fboaventura any news on this? Can I help test?

**Comment by fboaventura** [comment]
@ExquisiteMetropolis Yes and no. I went through the Home Assistant documentation but didn't have the time to create the needed configuration.

I need to make a few updates; I'll try to at least start this and ping you for testing, as I don't have a Home Assistant installation.

**Comment by awraeven** [comment]
@fboaventura thanks for the update. Let me know when I can give a hand in testing!

**Comment by fboaventura** [comment]
@ExquisiteMetropolis, can you please validate if it is working and reopen the issue if not?

**Comment by awraeven** [comment]
@fboaventura thanks for the effort! 👍 
I've tried to add this github repo to the Home Asssistant Addon Store.
(Ref: https://www.home-assistant.io/common-tasks/os#installing-a-third-party-add-on-repository)

The logs show an error on this part:
2025-04-09 08:35:56.348 INFO (MainThread) [supervisor.store.git] Cloning add-on /data/addons/git/f27c9bc8 repository from https://github.com/fboaventura/dckr-mrtg
2025-04-09 08:35:58.136 WARNING (SyncWorker_3) [supervisor.store.data] Can't read /data/addons/git/f27c9bc8/config.yaml: does not match regular expression ^([a-z0-9][a-z0-9.\-]*(:[0-9]+)?/)*?([a-z0-9{][a-z0-9.\-_{}]*/)*?([a-z0-9{][a-z0-9.\-_{}]*)$ for dictionary value @ data['image']. Got 'fboaventura/dckr-mrtg:v2.5.7'
value must be one of ['aarch64', 'amd64', 'armhf', 'armv7', 'i386'] @ data['arch'][1]. Got 'arm64v8'
`

(Github says I don't have rights to re-open this issue).

**Comment by fboaventura** [comment]
@ExquisiteMetropolis, Sorry about the false green. I oversaw some crucial parts of making this a Home Assistant Add-on.

For instance, to enable you to configure the settings, I need to change some parts of the starting script, to allow it to receive the information from Home Assistant.

Also, I'll have to change or add names to the built images, so Home Assistant can find them based on the architecture, as it will complain when I pass the image name without any `{arch}` marker. I'll try adding the `build.yml` instructions and see if it will work, in the meantime there is [this open issue](https://github.com/home-assistant/builder/issues/223) for the builder, to change and start reading from multi-arch images.

I'll need a bit more time to get this working.

**Comment by awraeven** [comment]
Surething! No rush. I'm happy to be able to help where I can on my part!
